app/assets/stylesheets/farmbot.css.scss

```diff
@@ -98,6 +101,46 @@ $brown:          #CA8;
   }
 }
 
+.designer-crop-info {
+  .crop-drag-info-image {
+    max-width: 98%;
+    padding: 0 auto 0 auto;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 auto` instead of `0 auto 0 auto`